scripts/test_solidity_lsp.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python3.9
```

Comment:
does it need 3.9?

Reply:
for type hints. I just realized I already reduced the kind of type hints I am using. Tested with 3.8 and that works, too.